index.html

```diff
@@ -56,6 +61,23 @@ <h4>Different types of alerts we came across in day today life</h4>
         </div>
 
     </section>
+    <h2>Badges</h2>
+    <section class="section bg-gray">
+        <h4>Badge on icon</h4>
+        <div class="badge-notification">
+            <i class="fa fa-envelope mail-icon"></i>
+        </div>
+
+        <div class="badge-notification">
+            <i class="fa fa-shopping-cart mail-icon"></i>
+        </div>
+        
+        <h4>Badge on Avatar</h4>
 
+        <div class="badge-notification">
+            <img src="images/default-avatar.svg" alt="default-Avatar" class="avatar-large">
```

Comment:
just a suggestion not mandatory that avatar-large could be avatar-lg a shorter class name try to give smaller class name would be better